apps/sms/test/marionette/lib/views/thread.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var View = require('./view');
+
+function Thread() {
```

Comment:
As I said above, I'd rather have all selectors inside Conversation view (extracted from ```lib/messages.js``` to ```lib/views/conversation.js```) so it's easier to track where selectors and methods come from and makes "view" more self contained. Having said that I think ```view.js``` and all this inheritance thing brings more complexity for no value.